src/FsCheck/Gen.fs

```diff
@@ -181,7 +181,7 @@ module Gen =
         | :? array<_> as arr ->
             elementsArr arr
         | _ ->
-            choose (0, (Seq.length xs)-1) |> map (flip Seq.nth xs)
+            choose (0, (Seq.length xs)-1) |> map (flip Seq.item xs)
```

Comment:
As you've noticed, this fails because master is still on an older version of FSharp.Core. :)